class-bookrestcontroller.php

```diff
@@ -0,0 +1,213 @@
+<?php
+
+class BookRestController extends WP_REST_Controller {
+	private $meta_key;
```

Comment:
comment

Reply:
If a look into the `__constructor` and how this property is used I'd rename this prop into `has_cap`